src/apps/Scripts/BaseScripts.js

```diff
@@ -66,6 +66,22 @@ const baseScripts = {
       # )
 
       # print (new_user)`
+  },
+  base_php: {
+    source: `
+      // Start coding!
+      // Note: You do not need opening / closing PHP tags in this script
```

Comment:
On smaller resolution it wrap and thus missing space between comment and code 
<img width="384" alt="screen shot 2016-10-21 at 17 30 57" src="https://cloud.githubusercontent.com/assets/11082340/19604113/265ee80e-97b4-11e6-8b95-54f2b9eb9edd.png">